client/remote.cpp

```diff
@@ -586,6 +586,12 @@ static int build_remote_int(CompileJob &job, UseCSMsg *usecs, MsgChannel *local_
                 throw remote_error(102, "Error 102 - command needs stdout/stderr workaround, recompiling locally");
             }
 
+            if (!crmsg->err.empty() && crmsg->err.find("file not found") != string::npos) {
```

Comment:
the emptiness check is superfluous.

Reply:
Removed in the just pushed patch. Thanks!